### DISCUSSION TITLE
Rapid v2.3 release

### DISCUSSION THREAD

**Discussion by bhousel** [facebook/Rapid#1414] [Show and tell]
I’m excited to announce that today we released **Rapid v2.3** editor for OpenStreetMap! The Rapid team has been busy working on this for the past few months - here’s 4 new features that we hope you’ll love…

## 🔙 Esri Wayback imagery 
The world never stops changing! We added a new background layer called "Esri Wayback" to let you view historical releases of the Esri World Imagery. This background source also has a date picker to help you choose from dates likely to show changes in the current map view. Open the Background pane to try it out!

![Esri Wayback](https://github.com/facebook/Rapid/assets/38784/a712466f-37b0-4a5b-86e2-3bade4525f9c)

## 🔄 Map Rotation
Take Rapid for a spin! Now you can rotate the map to make your editing easier (and avoid straining your neck!). Rapid supports rotating with <kbd>option</kbd>+click-drag or <kbd>shift</kbd>+<kbd>←</kbd>/<kbd>→</kbd> arrows, just like other maps you’ve used before. A new bearing control displays a north arrow, and you can click it to reset the map back to North-up. 

https://github.com/facebook/Rapid/assets/38784/ddf7fb65-3d0b-46be-b810-a54e78542d0d

## ✅ MapRoulette Integration
You can now complete MapRoulette tasks without ever leaving Rapid.  View active MapRoulette tasks in the current map view, or filter the results by challenge id.  After making your edits, select the task and click "I fixed it!", "Can't Complete", "Already Fixed", and "Not an Issue".  When you save your edits, your changeset will automatically include a description about any MapRoulette tasks that you've completed.

<img width="1143" alt="MapRoulette" src="https://github.com/facebook/Rapid/assets/38784/7c6ca4ab-0792-42cd-a875-d4fc3a7be4d1">

## 🎨 GeoScribbles
GeoScribble is a new service that allows mappers to take field notes while surveying with the EveryDoorOSM mobile app. Your sketches and notes will be waiting for you when you get back to your computer and open Rapid.

![GeoScribble](https://github.com/facebook/Rapid/assets/38784/33294d41-136b-424f-8a1c-b027c0a0423e)

---

We’ve got lots more planned, and we’re excited to hear how the community is using Rapid to map more efficiently.  
☝️ Make Rapid your OSM editor!  Bookmark https://rapideditor.org/edit and make it your daily mapping tool of choice.
✌️ Want to help us make Rapid better? Follow us on Github at https://github.com/facebook/Rapid or find us on any of the various social media or OpenStreetMap community channels.  We’d love to hear your thoughts!

Happy Mapping!  👍

**Comment by kleenkanteen** [comment]
Very cool

@bhousel How does Rapid communicate to OSM? Through the API documented here: https://wiki.openstreetmap.org/wiki/API_v0.6? It doesn't look like it has enough methods to allow all the web editor functinality though?

**Reply by bhousel** [reply]
Yes Rapid is using the OSM API.. Can you elaborate more which missing functionality you are looking for?

**Reply by kleenkanteen** [reply]
Nothing's missing, I was just doing a quick search of the API wiki and it didn't look like it had that many endpoints to support all the features of the editor. My estimation was wrong though I guess....
